main.py

```diff
@@ -1 +1,107 @@
-x = 50
+import pandas as pd
+import pyodbc
+
+def database_connect(server, database):
+    try:
+        conn = pyodbc.connect(
+        Trusted_Connection='Yes',
+        Driver='{ODBC Driver 17 for SQL Server}',
+        Server=server,
+        Database=database
+        )
+        cursor = conn.cursor()
+        return conn, cursor
+
+    except Exception as e:
+        print(f'Error connecting to the database: {e}')
+        return None, None
+
+def create_dbtable(original_file,cursor, conn):
+    try:
+        # Read the original csv data
+        masterlist = pd.read_csv(original_file, delimiter=';')
+        masterlist['Date'] = pd.to_datetime(masterlist['Date'])
+        #Create table in database
+        cursor.execute('''
+        CREATE TABLE masterlist (
+            Date datetime,
+            Server nvarchar(200),
+            Cost float
+            )
+        ''')
+        conn.commit()
+        print('Table is Created!')
+
+    except Exception as e:
+        print(f'Error creating table: {e}')
+
+def load_original_data(original_file, cursor, conn):
+    try:
+        masterlist = pd.read_csv(original_file, delimiter=';')
+        for row in masterlist.itertuples():
+            cursor.execute(
+                '''
+                INSERT INTO [Python-SQL-CSV].[dbo].[masterlist]
+                (Date,Server,Cost)
+                VALUES(?, ?, ?)
+                ''',
+                row.Date,
+                row.Server,
+                row.Cost
+            )
+        conn.commit()
+        print('Original Data is loaded.')
+
+    except Exception as e:
+        print(f"Error loading data: {e}")
+
+def load_incremental_data(new_file, cursor, conn):
+    try:
```

Comment:
this try block is too large, which many things can fail, and everything is being caught with the general Exception type. Extract what can not fail (or is unlikely to fail) from the try block (for example the pandas stuff